lib/nose/backend/cassandra.rb

```diff
@@ -312,7 +312,7 @@ def cql_where_clause(conditions)
           where = @eq_fields.map do |field|
             "\"#{field.id}\" = ?"
           end.join ' AND '
-          unless @range_field.nil?
+          unless @range_fields.to_a.empty?
             condition = conditions.each_value.find(&:range?)
             where << " AND \"#{condition.field.id}\" #{condition.operator} ?"
```

Comment:
Wouldn't this need multiple conditions (more than one `AND`) since `condition.field` is now an array? (see my previous comment)